core/trino-main/src/main/java/io/trino/operator/SimplePagesHashStrategy.java

```diff
@@ -39,7 +39,7 @@
 {
     private static final int INSTANCE_SIZE = ClassLayout.parseClass(SimplePagesHashStrategy.class).instanceSize();
```

Comment:
`SimplePagesHashStrategy` is only used when `JoinCompiler` pages hash strategy compilation fails. Shouldn't `io.trino.sql.gen.JoinCompiler` be fixed too?

Reply:
Sounds reasonable. let's have an issue to ensure followup

Reply:
Created https://github.com/trinodb/trino/issues/8177